proposals/simd/SIMD.md

```diff
@@ -665,6 +665,11 @@ natural alignment.
 
 Load a `v128` vector from the given heap address.
 
+```python
+def S.load(memarg):
+    return S.New(memory.bytes[memarg.start:memarg.start + 16])
```

Comment:
`S.New` is still ambiguous, so how about using a for loop over the lanes using `S.LaneType.Load`?

Reply:
Yeah, that is good point, changing to `from_bytes`, which seems to be a standard-ish way to do this in Python